docs/usage/validation_errors.md

```diff
@@ -394,7 +396,7 @@ except ValidationError as exc_info:
 
 ## `datetime_object_invalid`
 
-This error is raised when the datetime object is invalid.
+This error is raised when the `datetime` object is not valid.
```

Comment:
```suggestion
This error is raised when something about the `datetime` object is not valid.
```
This is sort of weird, I think the tweaked phrasing helps clarify that it's intended as a bit of a catch-all?

(The example below demonstrates it can be raised for invalid timezones, for example)